sdk/monitor/azure-monitor-opentelemetry-autoconfigure/src/main/java/com/azure/monitor/opentelemetry/autoconfigure/implementation/quickpulse/filtering/KnownDependencyColumns.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.monitor.opentelemetry.autoconfigure.implementation.quickpulse.filtering;
+
+public class KnownDependencyColumns {
```

Comment:
An Enum type could perhaps be used. The advantage with this type is that it could allow later the addition of methods / behaviors specific to an enum value (which is not possible with a constant).

Reply:
I considered an enum, but really just need a way to have string constants for the column names; not sure what other functionality of enum would be useful for this PR